appsec-kit-v8/src/main/java/com/vaadin/appsec/v8/ui/content/MainView.java

```diff
@@ -31,6 +34,9 @@
  */
 public class MainView extends AbstractAppSecContent {
```

Comment:
To make the logger name meaningful, I suggest to rename this class to `AppSecView` as we have in the V24 module.

Reply:
Good point. Renamed.